### ISSUE TITLE
Syntax highlighting scheme should pass WCAG 2.0 AAA ratings

### ISSUE THREAD

**Issue by kewisch** [mozilla/addons-code-manager#774]
The color for a text file without syntax highlighting is `#657b83` on `#FFFFFF`, which gives you a contrast ratio of 4.45:1. For people looking at code all day it would be great to use a color scheme that is as good as possible on the eyes, so aim for an AAA rating of at least 7:1.

See https://www.w3.org/TR/WCAG/#contrast-enhanced for details.

**Comment by kumar303** [comment]
https://color.review/ would be a good tool to use while working on this bug

**Comment by mirefly** [comment]
Simply using darker text color leads to lower contrast ratio of selected code (on black background). `#657b83` (base00 in Solarized Color Schemes) gives a contract ratio of 4.4 on white background, and 4.7 on black (when the code is selected); `#586e75` (base01) gives 5.3 on white, and 3.9 on black. 

I guess that we should change both text color and background for selected code. (We only change background now). https://github.com/mozilla/addons-code-manager/blob/83773190ae79e6186656b0ac2bcfb937faa5bc02/src/refractor/prism.css#L55
However, to make it works properly, I have to list many classes whose color has been set. I am not sure if I should do this.
```css
pre[class*="language-"]::-moz-selection,
pre[class*="language-"] not(***list many class like .token.entity ***) ::-moz-selection {
  color:white;
  backgroud: black;
}
```
@kumar303 Thanks for letting me know this issue.

**Comment by kumar303** [comment]
@mirefly does it help if you limit it to just the [`text`](https://github.com/mozilla/addons-code-manager/blob/a809064e89bf8c1ddb6ab5046d99a55e33a71fa9/src/refractor/index.tsx#L8) language? I think this will limit the selector to `text` but I'm not sure: `pre[class="language-text"]`

**Comment by mirefly** [comment]
@kumar303 Thanks!  `language-text` makes it much easier.

**Comment by kumar303** [comment]
Thanks again for your help, @mirefly. I've tried to find some other good first bugs but there aren't a lot. I'll add labels as I see them: https://github.com/mozilla/addons-code-manager/issues?q=is%3Aissue+is%3Aopen+label%3A%22contrib%3A+good+first+bug%22

**Comment by ioanarusiczki** [comment]
I see the changes on https://code.addons-dev.allizom.org 
it's pre[class="language-text"] { color: #073642; } instead of color: #657b83;

Before - on AMO stage 

![AMO stage](https://user-images.githubusercontent.com/33448286/59275928-f87c6c00-8c65-11e9-90ca-70702f7b0ea3.png)

After - on AMO dev 
![AMO dev](https://user-images.githubusercontent.com/33448286/59276309-bdc70380-8c66-11e9-96d0-894136e64b7b.png)
